ppr-ui/src/components/parties/composables/useSecuredParty.ts

```diff
@@ -96,21 +96,28 @@ export const useSecuredParty = (props, context) => {
   }
 
   const hasMatchingSecuredParty = (addedParty: PartyIF): boolean => {
-    // store state without newly added party
+    // store state without newly added party.
     const parties = cloneDeep(getAddSecuredPartiesAndDebtors.value.securedParties)
     if (localState.partyType === SecuredPartyTypes.INDIVIDUAL) {
       return parties.some(party =>
-        isEqual(party.personName, addedParty.personName) &&
-        isEqual(party.address, addedParty.address)
+        isObjectEqual(party.personName, addedParty.personName) &&
+        isObjectEqual(party.address, addedParty.address)
       )
     } else {
       return parties.some(party =>
-        party.businessName === addedParty.businessName &&
-        isEqual(compact(party.address), compact(addedParty.address))
+        party.businessName?.toUpperCase() === addedParty.businessName?.toUpperCase() &&
+        isObjectEqual(party.address, addedParty.address)
       )
     }
   }
 
+  const isObjectEqual = (object1: any, object2: any): boolean => {
+    const workObject1 = Object.create(object1)
+    const workObject2 = Object.create(object2)
+    const addressEqual = isEqual(normalizeObject(workObject1), normalizeObject(workObject2))
+    return addressEqual
+  }
+
```

Comment:
As mentioned in the first comment, thinking this is also a utility function. 
Besides the address const name it could be used for anything, though not a big deal, can be relocated in a future commit as well, whatever you prefer.

Reply:
Agree with Cameron, this should be moved out to a util file.

Reply:
Ok